apps/bot/lib/bot/schedulers/github_issues.ex

```diff
@@ -34,8 +35,19 @@ defmodule Bot.Scheduler.GitHubIssues do
     {:noreply, new_state}
   end
 
+  def handle_info(:project_database_sync, state) do
+    new_state =
+      state
+      |> Map.merge(init_state())
```

Comment:
We don't need to pipe this since it's only once: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines

All you need here is:
```elixir
new_state = Map.merge(state, init_state())
```

Reply:
Thank you for the feedback, rebased and squashed!